i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/load-aware-scheduling.md

```diff
@@ -1,45 +1,37 @@
-# Load Aware Scheduling
+# 负载均衡调度
 
-Load Aware Scheduling is an ability of koord-scheduler for balancing pods scheduling based on the real-time load of each node.
+Load Aware Scheduling 是 koord-scheduler 根据每个节点的实时负载平衡 Pod 调度的能力.
 
-## Introduction
+## 简介
 
-Load balancing is a common issue in resource scheduling. Under-utilized nodes bring much resource waste to the
-cluster, while over-utilized nodes are likely to cause performance degradation. Neither of them is suitable for
-efficient resource management.
+负载均衡是资源调度中的常见问题. 未充分利用的节点会给集群带来很大的资源浪费, 而过度使用的节点可能会导致性能下降. 他们都不是有效的资管管理.
 
-The native Kubernetes scheduler schedules pods based on the requests and the allocation of nodes, considering neither
-the real-time load nor the estimated usage. When we want to balance the pod scheduling on each node and make the loads
-even with the native scheduler, we need to set precise resource requirements for the applications. Moreover, since
-Koordinator enables resource overcommitment to achieve better resource efficiency, we need a mechanism to reduce the
-probability of performance degradation and avoid over-utilization.
+原生 Kubernetes scheduler 调度程序根据请求和节点分配来调度 Pod，既不考虑实时负载，也不考虑估计使用量。 当我们想要平衡每个节点上的 Pod 调度，使负载与原生调度器相同时，我们需要为应用程序设置精确的资源需求。 此外，由于 Koordinator 启用资源过度使用以实现更好的资源效率，我们需要一种机制来降低性能下降的概率并避免过度使用。
```

Comment:
Kubernetes scheduler 已经是调度器的意思了。后面中文的”调度程序“是可以考虑去掉的。

Reply:
"当我们想要平衡每个节点上的 Pod 调度，使负载与原生调度器相同时" .

When we want to balance the pod scheduling on each node and make the loads
even with the native scheduler, we need to set precise resource requirements for the applications. 

当我们期望使用原生调度器均匀的打散Pod并保持节点间的负载均衡，我们需要为应用设置精确的资源规格。

Reply:
Moreover, since
Koordinator enables resource overcommitment to achieve better resource efficiency, we need a mechanism to reduce the
probability of performance degradation and avoid over-utilization.

此外，当 Koordinator 通过超卖机制提升资源使用效率时，我们需要一种机制尽量避免性能回退，并避免负载过高的问题。